scipy/spatial/_voronoi.pyx

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+cimport numpy as np
+cimport cython
+
+__all__ = ['sort_vertices_of_regions']
+
+cdef void remaining_filter(np.npy_intp shape, np.npy_intp[:] remaining,
+		np.npy_intp current_simplex):
+   cdef np.npy_intp i
+   for i in range(shape):
+     if remaining[i] == current_simplex:
+       remaining[i] = -2
+    
+@cython.boundscheck(False)
+def sort_vertices_of_regions(int[:,::1] simplices, regions):
+  cdef np.npy_intp n, k, s, i
```

Comment:
indentation is 2 spaces here, and 3 spaces in the function above. can you make it 4 everywhere?